content/forgero-compat/src/main/resources/assets/forgero/lang/en_us.json

```diff
@@ -243,5 +243,8 @@
   "item.forgero.netherite_gold_ingot": "Netherite-Gold Ingot",
   "item.forgero.netherite_emerald_ingot": "Netherite-Emerald Ingot",
   "item.forgero.netherite_diamond_ingot": "Netherite-Diamond Ingot",
-  "item.forgero.netherite_iron_ingot": "Netherite-Iron Ingot"
+  "item.forgero.netherite_iron_ingot": "Netherite-Iron Ingot",
+  "item.forgero.obsidian_stem": "Obsidian Stem",
+  "item.forgero.reinforced_crimson_planks": "Reinforced Crimson Planks",
```

Comment:
Missing entry for Reinforced crimson:
![image](https://github.com/user-attachments/assets/fe06ea89-a382-454e-82e1-3c9c1ba56fb9)

Reply:
Same with warped

Reply:
I assume the _planks should be removed and that should be good.